src/runtime_src/core/pcie/windows/alveo/CMakeLists.txt

```diff
@@ -34,6 +34,7 @@ add_library(xrt_core_static STATIC
 target_link_libraries(xrt_core
   PRIVATE
   xrt_coreutil
+  setupapi
```

Comment:
This link library is included as a pragma already.

Reply:
This is true. The pragmas are not used by MinGW, so I have to link them differently. However, these lines should depend on the used compiler. I updated the files accordingly to only use them if the compiler is GNU.